hyperstyle/src/python/review/reviewers/perform_review.py

```diff
@@ -99,7 +99,11 @@ def perform_review(path: Path, config: ApplicationConfig) -> GeneralReviewResult
 
 def _preform_review(metadata: Metadata, languages: List[Language], config: ApplicationConfig) -> GeneralReviewResult:
     # TODO start review for several languages and do something with the results
-    reviewer = language_to_reviewer[languages[0]]
+    if config.inspectors_config['language'] == Language.UNKNOWN:
```

Comment:
By default you use `None`, why here do you use` Language.UNKNOWN`?

Reply:
Also, this construction looks strange, I'd prefer to use only one place for such operations because earlier you will try to guess a language, but next you check the `inspectors_config`, and if it contains any language you ignore the work which has already been done. Probably it is better to move this check into the `perform_review` function, or (it is better) create a new field language in `ApplicationConfig` and calculate the language before calling the `perform_review` function. What do you think?

Reply:
Yes, I moved language to ApplicationConfig. To do that LanguageVersion had to move to separated module for avoiding circular imports. 

I didn't disable exploring projects if language explicitly set, because there is a metadata fetching that cannot be skipped.

Also can we sort detected languages to disable returning randomized results? This temporary solution will help to get predictable results unless multiple languages support will implement.